.ci/deps.sh

```diff
@@ -61,3 +62,6 @@ wget "https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.0.1/ast
 tar -xvzf ~/astyle.tar.gz -C ~/
 make -C ~/astyle/build/gcc
 sudo make install -C ~/astyle/build/gcc
+
+# gometalinter installation
+gometalinter --install
```

Comment:
this is to be added here, right?

Reply:
Copied this to https://github.com/coala/coala-bears/pull/2536